### ISSUE TITLE
Uncaught Error: `path\to\node_modules\fibers\bin\win32-x64-v8-4.7\fibers.node` is missing

### ISSUE THREAD

**Issue by userabuser** [laverdet/node-fibers#276]
Hi...

Requiring `fibers` results in the following error:

`Uncaught Error: path\to\node_modules\fibers\bin\win32-x64-v8-4.7\fibers.node is missing`

**win32-x64-v8-4.7** does not exist within bin; my v8 version `v8: "4.7.80.27"`

I attempted to clone the repo and build however that fails for me, I've included the `npm-debug.log` file that includes more verbose reporting.

[npm-debug.log.txt](https://github.com/laverdet/node-fibers/files/192878/npm-debug.log.txt)

Any suggestion on how to resolve the build process as I couldn't determine what, if any issue there was/is by inspecting `build.js` and or will you be updating the npm package at some point to cater for the above?

Thank you.

**Comment by sjmcdowall** [comment]
Unless that exact node version is critical -- the easiest solution would be to use a V8 version that is already compiled and available as part of the fibers package?  

If you really need to have that version and compile it -- do you have a VS 2012 C++ development environment including command line build tools installed?  I think that's the requirement to compile fibers on Windows..

**Comment by laverdet** [comment]
Are you running a custom build of node? The correct v8 version for node 4.4.1 (which is what I see in your npm-debug.log) is 4.5.103.

**Comment by userabuser** [comment]
@laverdet actually, I neglected to mention that I am running `node-fibres` in Electron `0.36.11` which of course is running node `5.1.1` and v8 `4.7.80.27`. I didn't catch on to the fact until I attempted to debug from CLI and saw that node was `4.4.1`. 

![facepalm](https://cloud.githubusercontent.com/assets/1643834/14170428/4331b8ce-f775-11e5-980a-6a3585b24d31.gif)

@sjmcdowall I have VS C++ redistributables spanning 2008 to 2013, but only have VS 2010 Express Edition/CLI tools. I'll aquire 2012 and see how I go.

---
Thank for your help guys... can probably close out this ticket, I'll try and work out the build for 5.1.1 myself.
